src/web/app/src/components/SignUp/Forms/BlogFeeds.tsx

```diff
@@ -7,7 +7,9 @@ const BlogFeeds = () => {
   return (
     <RSSFeeds
       title="Blog and RSS"
-      prompt="Enter your blog URL and select the RSS you want to use in Telescope"
+      prompt="Enter your blog URL and select the RSS you want to use in Telescope.
+      A blog URL (HTML) should be the root of your blog itself, and Telescope will find the link(s) of blog feed URL (XML) for you to select, such as:"
```

Comment:
Really picky thing, but I think you should add `the` before the blog
```suggestion
      A blog URL (HTML) should be the root of your blog itself, and Telescope will find the link(s) of the blog feed URL (XML) for you to select, such as:"
```

Reply:
Thanks, but it won't apply after the new changes.